app/assets/stylesheets/tags_input.scss

```diff
@@ -0,0 +1,59 @@
+.tag-words-guide {
+  margin-left: 3rem;
+  font-size: 0.8rem;
+  color: $input-border-color
+}
+
+label.tags-input.active {
+  top: 0.2rem;
+}
+
+div.for-tags-input {
+  margin-top: 2.5rem;
+}
+
+div.tagsinput {
+  // Adjust
+  margin: 0 0 30px 3rem;
```

Comment:
Properties should be ordered background-color, font-size, height, margin, outline, overflow-y, width